src/Nfield.Quota/QuotaVariableDefinition.cs

```diff
@@ -21,5 +23,38 @@ public QuotaVariableDefinition(IEnumerable<QuotaLevelDefinition> levels)
         public string OdinVariableName { get; set; }
 
         public ICollection<QuotaLevelDefinition> Levels { get; }
+
+        public static bool operator ==(QuotaVariableDefinition left, QuotaVariableDefinition right)
+        {
+            return left?.Equals(right) ?? false;
+        }
+
+        public static bool operator !=(QuotaVariableDefinition left, QuotaVariableDefinition right)
+        {
+            return !(left == right);
+        }
+
+        public override bool Equals(object obj)
+        {
+            var other = obj as QuotaVariableDefinition;
+            return other != null && Equals(other);
```

Comment:
`other != null` always returns `true`, so remove it or use `Object.ReferenceEquals(other, null)` instead